esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/ppd42ns-wificonfig-ppd-sds-dht.ino

```diff
@@ -609,36 +616,41 @@ void readConfig() {
 				if (json.success()) {
 					debug_out(F("parsed json..."),DEBUG_MIN_INFO,1);
 					if (json.containsKey("SOFTWARE_VERSION")) strcpy(version_from_local_config, json["SOFTWARE_VERSION"]);
```

Comment:
Then please add this explanation as a comment to the declaration of the variable.